app/views/layouts/_layout.njk

```diff
@@ -35,12 +36,49 @@
     {% endblock %}
 
     {% block header %}
-      {{ govukHeader({
-        homepageUrl: "https://www.gov.uk/",
-        containerClasses: "govuk-width-container",
-        serviceName: serviceName,
-        serviceUrl: "/"
-      }) }}
+      {% if user %}
+        {{ mojHeader({
```

Comment:
Could this be added as a separate header.njk file?

Reply:
yup.